### PR TITLE
Add saveData attribute to the NetInfo samples page

### PR DESCRIPTION
**Pull request by ghost**
Add saveData attribute to the NetInfo samples page

Fixes https://github.com/GoogleChrome/samples/issues/560

### CODE REVIEW

**Comment by ghost** [comment]
@beaufortfrancois can you PTAL. I think I am not a repo owner. So, may be you need to merge this and somehow tie it to https://github.com/GoogleChrome/samples/issues/560?

File: network-information/index.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 feature_name: Network Information API
-chrome_version: 62
+chrome_version: 65
```
Comment (ebidel) [inline network-information/index.html:3]: @jeffposnick what's our precedent for updating the version in a demo? If a feature is amended later...?

Comment (jeffposnick) [reply network-information/index.html:3]: I think updating it with the correct minimum version to use the entirety of the sample is the precedent. That's what happened in https://github.com/tarunban/samples/commit/d3254545c43f86d5d1b102bd857836abaf066a26#diff-a318373df9ff649fbbc1264ae0e29971

**Review by ebidel — APPROVED** [review]

**Comment by ghost** [comment]
Can somebody merge this pull request? I do not have write access to this repo.

**Comment by beaufortfrancois** [comment]
Thank you @tarunban!
You can try it now at https://googlechrome.github.io/samples/network-information/

**Pull request merged**